app/controllers/topological_inventory/ingress_api/v0/inventory_controller.rb

```diff
@@ -5,9 +5,17 @@ module IngressApi
     module V0
       class InventoryController < ApplicationController
         skip_before_action :verify_authenticity_token
+
         def save_inventory
           payload = JSON.parse(request.body.read)
 
+          root = OpenAPIParser.parse(JSON.parse(File.read('public/doc/openapi-3-v0.0.2.json')),
```

Comment:
There should be a better way to get this via the OpenApi::Docs class, so we don't have to hardcode paths.

Reply:
Ah, and that should already be loaded in memory, right? So we don't load it from file everytime

Reply:
hm, I think we will also want to cache the `OpenAPIParser.parse` since that is the parsers internal representation